packages/proxy/lib/http/request-middleware.ts

```diff
@@ -242,32 +392,60 @@ function reqNeedsBasicAuthHeaders (req, { auth, origin }: Cypress.RemoteState) {
 }
 
 const MaybeSetBasicAuthHeaders: RequestMiddleware = function () {
+  const span = telemetry.startSpan({ name: 'maybe:set:basic:auth:headers', parentSpan: this.reqMiddlewareSpan, isVerbose })
+
   // get the remote state for the proxied url
   const remoteState = this.remoteStates.get(this.req.proxiedUrl)
 
-  if (remoteState?.auth && reqNeedsBasicAuthHeaders(this.req, remoteState)) {
+  const doesReqNeedBasicAuthHeaders = remoteState?.auth && reqNeedsBasicAuthHeaders(this.req, remoteState)
+
+  span?.setAttributes({
+    doesReqNeedBasicAuthHeaders,
+  })
+
+  if (remoteState?.auth && doesReqNeedBasicAuthHeaders) {
     const { auth } = remoteState
     const base64 = Buffer.from(`${auth.username}:${auth.password}`).toString('base64')
 
     this.req.headers['authorization'] = `Basic ${base64}`
   }
 
+  span?.end()
   this.next()
 }
 
 const SendRequestOutgoing: RequestMiddleware = function () {
+  // end the request middleware here before we make
+  // our outbound request so we can see that outside
+  // of the internal cypress middleware handlers
+  this.reqMiddlewareSpan?.end()
+
+  // the actual req/resp time outbound from the proxy server
+  const span = telemetry.startSpan({
+    name: 'outgoing:request:ttfb',
+    parentSpan: this.handleHttpRequestSpan,
+    isVerbose,
+  })
+
   const requestOptions = {
+    browserPreRequest: this.req.browserPreRequest,
     timeout: this.req.responseTimeout,
     strictSSL: false,
     followRedirect: this.req.followRedirect || false,
     retryIntervals: [],
     url: this.req.proxiedUrl,
+    time: true, // include timingPhases
```

Comment:
I wonder if we want to only pass this in if verbose telemetry is enabled, since there might be a performance hit on doing the math operations in [@cypress/request](https://github.com/cypress-io/request/blob/master/request.js#L705)

Reply:
I'll make it conditional on the span being present, that should cover any scenario.